### DISCUSSION TITLE
DB/db4/

### DISCUSSION THREAD

**Discussion by giscus** [HobbitQia/notebook#24] [General]
# DB/db4/

HobbitQia的笔记本

https://note.hobbitqia.cc/DB/db4/

**Comment by catastrophecaptain** [comment]
"如果不作为权限，我们可以通过间接的外键约束和 cascade 删掉被引用的数据。（删掉饮用者，则被引用者也要被删除）因此这也是个权限"

**Reply by catastrophecaptain** [reply]
这里引用和被引用写反了吧
